packages/runtime-dom/src/components/TransitionGroup.ts

```diff
@@ -170,8 +175,18 @@ function callPendingCbs(c: VNode) {
   }
 }
 
+function getRelativePosition(el: Element): Position {
+  const elRect = el.getBoundingClientRect()
+  const parentRect = el.parentElement?.getBoundingClientRect()
+
+  return {
+    left: elRect.left - (parentRect?.left || 0),
+    top: elRect.top - (parentRect?.top || 0)
```

Comment:
There's a project convention of avoiding optional chaining (`?.`) in client-side code because of the impact on bundle size. An ESLint rule was recently added to enforce this: #10919.